src/core/net_db.cc

```diff
@@ -134,26 +136,31 @@ void NetDb::Run() {
       if (!m_IsRunning)
         break;
       uint64_t ts = i2p::util::GetSecondsSinceEpoch();
-      if (ts - last_manage_request >= 15) {  // manage requests every 15 seconds
+      if (ts - last_manage_request >=
+          static_cast<std::size_t>(NetDbDuration::ManageRequestsInterval)) {  // manage requests every 15 seconds
```

Comment:
We can go ahead and play with the documentation. For us to specify an interval in comments defeats what we're trying to achieve here. I'd suggest to move the comment above the `if` block and describe what will happen within the block and *why* it will happen. This is general documentation advice (and documenting *why* is the fun part :smile:)